File: utils/utils-base/src/com/xenoage/utils/font/TextMeasurer.java
package com.xenoage.utils.font;

/**
 * This interfaces provides methods to measure Strings.
 * 
 * @author Andreas Wenger
 */
public interface TextMeasurer {

	/**
	 * Measures the given text, using the given {@link FontInfo}.
	 */
	public TextMetrics measure(FontInfo font, String text);

}

File: utils-kotlin/src/com/xenoage/utils/font/TextMeasurer.kt
package com.xenoage.utils.font

/**
 * This interfaces provides methods to measure strings
 * with a given [FontInfo].
 */
interface TextMeasurer {

	/** Measures the given text, using the given [FontInfo]. */
	fun measure(font: FontInfo, text: String): TextMetrics

}
